cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js

```diff
@@ -0,0 +1,168 @@
+import uuid from 'uuid';
+import TestTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import permissions from '../../support/dictionary/permissions';
+import UserEdit from '../../support/fragments/users/userEdit';
+import TopMenu from '../../support/fragments/topMenu';
+import generateUniqueItemBarcodeWithShift from '../../support/utils/generateUniqueItemBarcodeWithShift';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../support/fragments/users/users';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import getRandomPostfix, { getTestEntityValue } from '../../support/utils/stringTools';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import TitleLevelRequests from '../../support/fragments/settings/circulation/titleLevelRequests';
+import Requests from '../../support/fragments/requests/requests';
+import RequestDetail from '../../support/fragments/requests/requestDetail';
+import MoveRequest from '../../support/fragments/requests/move-request';
+
+describe('Title Level Request', () => {
+  let userData = {};
+  const patronGroup = {
+    name: getTestEntityValue('groupTLR'),
+  };
+  const instanceData = {
+    title: getTestEntityValue('InstanceTLR'),
+  };
+  const testData = {
+    userServicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation('autotestTLR', uuid()),
+  };
+
+  before('Preconditions', () => {
+    cy.getAdminToken()
+      .then(() => {
+        cy.loginAsAdmin({
+          path: SettingsMenu.circulationTitleLevelRequestsPath,
+          waiter: TitleLevelRequests.waitLoading,
+        });
+        ServicePoints.createViaApi(testData.userServicePoint);
+        testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+        });
+      })
+      .then(() => {
+        instanceData.itemsData = [
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: 'Available' },
```

Comment:
please add available item status and add it to the constants file: cypress\support\constants.js

Reply:
done